appveyor.yml

```diff
@@ -2,7 +2,6 @@ version: "{build}"
 
 environment:
   matrix:
-    - nodejs_version: "16"
     - nodejs_version: "18"
     - nodejs_version: "19"
```

Comment:
We can drop 19 support since it is no longer supported. Should we add 20 and 21 here?

Reply:
Yes, let's drop support for 19 and add 20 and 21.

Reply:
Just push a commit for adding 20, 21 and remove 19 👍 @arthurschreiber , the CI still expecting check for node 16 and 19, is this configurable from one of the yml files or it is a ci setting? 🤔